docs/octopus-cloud/index.md

```diff
@@ -51,21 +51,38 @@ If you are having difficulty uploading packages to your Octopus built-in package
 
 In the near future, the Octopus Cloud will become available in additional regions and you'll have the ability to move between regions.
 
-## Reset the User Password
+## Reset the Octopus Account Password
 
-If you forget your username or password for the instance, you can request a **new user invite**.
+If you forget your username or password for your Octopus account or Octopus instance, you can reset them via Octopus.com
 
-1. Log in to your Octopus account.
-1. Select your cloud instance.
-1. Click the **new user invite** link.
+1. Visit [Octopus.com/signin](https://Octopus.com/signin).
+1. Click 'Forgot your password?'
+1. Reset your credentials and log into your Octopus account. You can now launch your Octopus instance from the accounts dashboard.
+
+## Invite Users to your Octopus Cloud Instance
```

Comment:
I think this would generate an anchor for linking to from the product/other places
```suggestion
## Invite Users to your Octopus Cloud Instance {#OctopusCloud-Invitingusers}
```

Reply:
Adding that does indeed generate an anchor. Anchors are also generated from the title alone, but your anchor is much more elegant that the autogenerated one would be (i.e., #invite-users-to-your-octopus-cloud-instance).